src/main/java/com/stripe/net/StripeResponseGetter.java

```diff
@@ -18,6 +20,22 @@ <T extends StripeObjectInterface> T request(
       ApiMode apiMode)
       throws StripeException;
 
+  @SuppressWarnings("TypeParameterUnusedInFormals")
+  default <T extends StripeObjectInterface> T request(ApiRequest request, Type typeToken)
+      throws StripeException {
+    return request(
+        request.getBaseAddress(),
+        request.getMethod(),
+        request.getPath(),
+        request.getParams(),
+        typeToken,
+        request.getOptions(),
+        request.getApiMode());
+  };
+
+  /** @deprecated Use {@link #requestStream(ApiRequest, Type)} instead. */
```

Comment:
```suggestion
  /** @deprecated Use {@link #requestStream(ApiRequest)} instead. */
```
Sorry CI hated my suggestion 😅